src/storage/mongodb/ChargingStationStorage.ts

```diff
@@ -256,7 +256,7 @@ export default class ChargingStationStorage {
     // Connector Status
     if (params.connectorStatuses) {
       filters['connectors.status'] = { $in: params.connectorStatuses };
-      filters.inactive = false;
+      filters.inactive = true;
```

Comment:
Why is the inactive flag set to true, when we filter for connectorStatuses?
We will not get any results, if we search for e.g. status 'Available' right?

Reply:
I just pushed a temporary test, my bad